src/utils.c

```diff
@@ -1788,6 +1788,9 @@ void utils_tidy_path(gchar *filename)
 	const gchar *needle;
 	gboolean preserve_double_backslash = FALSE;
 
+	if (utils_is_uri(filename))
+		return;
```

Comment:
@b4n I kept it this way in the new patch - the current implementation of the function would have to be changed because there's e.g.

```
utils_string_replace_all(str, "/", G_DIR_SEPARATOR_S)
```

on Windows which we don't want. And since URIs cannot be relative, I think it's more or less OK the way it is?